### PR TITLE
Another Android+clang assembly build fix

### PR DESCRIPTION
**Pull request by grendello**
This time for clang from an older NDK (14b) which not only needs the previous
fix to enable the crypto extensions but it also needs NEON to be explicitly
enabled in order to avoid errors similar to:

    ghashv8-armx.S:14:2: error: instruction requires: neon
     movi v19.16b,#0xe1

### CODE REVIEW

**Comment by lewurm** [comment]
I assume you have tested that, right?

I'm a bit surprised, because `neon` is an instruction set extension for <= ARMv7.  It's also not listed in the GCC options for ARMv8:
https://gcc.gnu.org/onlinedocs/gcc/AArch64-Options.html (can't find the according docs for clang 😕 )

**Comment by grendello** [comment]
Yes, I tested it. The fix is for clang shipped with NDK r14, it's some version of clang 3 I think - probably a bug in it. `+neon` wasn't necessary for the NDK r18's clang version.

**Review by lewurm — APPROVED** [review]
https://xamarinhq.slack.com/archives/C03CELHQQ/p1546549800066900
TL;DR: untested with NDK r18, might need changes for that. We can go with this for now.
